tests/generic/check_dex_null_amount_usd.sql

```diff
@@ -0,0 +1,27 @@
+{% test check_dex_null_amount_usd(model) %}
+
+    {%- set threshold = 0.05 -%}
```

Comment:
This threshold is open to discuss.

Reply:
Maybe `0.16`.

Reply:
I will change to 0.50 first.
```suggestion
    {%- set threshold = 0.50 -%}
```